goovn/uuid.go

```diff
@@ -0,0 +1,13 @@
+package goovn
+
+import (
+	"github.com/google/uuid"
+)
+
+func newUUID() (string, error) {
```

Comment:
what is this for?

Reply:
I'm change from rand to proper uuid, i think that is more generic and have less ability to duplicate..